src/dune_rules/artifact_substitution.ml

```diff
@@ -52,8 +52,7 @@ type conf =
   }
 
 let mac_codesign_hook ~codesign path =
-  Process.run ~display:!Clflags.display Strict codesign
-    [ "-s"; "-"; Path.to_string path ]
+  Process.run ~display:Quiet Strict codesign [ "-s"; "-"; Path.to_string path ]
```

Comment:
When we are code signing on macos, we need to run an external program internally. We do not need to report this to the user.